inst/sql/sql_server/field_is_standard_valid_concept.sql

```diff
@@ -54,6 +54,7 @@ FROM
   		JOIN @cohortDatabaseSchema.@cohortTableName c 
   		    ON cdmTable.person_id = c.subject_id
   		    AND c.cohort_definition_id = @cohortDefinitionId
-    }
+    	}
+	WHERE cdmTable.@cdmFieldName IS NOT NULL
```

Comment:
this change removes overlap between this check and isRequired / standardConceptRecordCompleteness